prwkv/rwkvrnnmodel.py

```diff
@@ -262,6 +265,10 @@ def generate(self,
 
                 next_token = token_id
                 
+            if update_state:
```

Comment:
Not sure why we would have this here in the api, for the generate use case, we want it to just generate using what ever initialized state simplifying the work flow. Lmk what you think